src/graphql/data/Token.ts

```diff
@@ -48,23 +48,24 @@ export const tokenQuery = graphql`
         twitterName
         logoUrl
         tokens {
-          chain @required(action: LOG)
-          address @required(action: LOG)
+          chain
+          address
         }
       }
     }
   }
 `
-export type { Chain, TokenQuery } from './__generated__/TokenQuery.graphql'
+// eslint-disable-next-line import/no-unused-modules
+export type { Chain, TokenQuery } from './__generated__/types-and-hooks'
 
-export type TokenQueryData = NonNullable<TokenQuery$data['tokens']>[number]
+export type TokenQueryData = NonNullable<TokenQuery['tokens']>[number]
 
 // TODO: Return a QueryToken from useTokenQuery instead of TokenQueryData to make it more usable in Currency-centric interfaces.
 export class QueryToken extends WrappedTokenInfo {
   constructor(data: NonNullable<TokenQueryData>, logoSrc?: string) {
     super({
       chainId: CHAIN_NAME_TO_CHAIN_ID[data.chain],
-      address: data.address,
+      address: data.address ?? '',
```

Comment:
The real solution here would be to have data team update the schema to make this field non-nullable instead of enforcing non-null values on the FE.

Reply:
Still, I'm afraid this might cause a regression, because `address` is assumed to be valid here.
At the callsite in TokenDetails L77, address is checked to be defined. Could you just pass that address directly into the constructor (add it as a first parameter)?

Reply:
Okay I'm passing in the address and use it as the fallback in case data.address is undefined